source/Calamari/Kubernetes/Conventions/Helm/HelmManifestAndStatusReporter.cs

```diff
@@ -47,43 +47,37 @@ await Task.Run(async () =>
                            {
                                var resourceStatusCheckIsEnabled = deployment.Variables.GetFlag(SpecialVariables.ResourceStatusCheck);
 
-                               if (
-                                   resourceStatusCheckIsEnabled
```

Comment:
I've actually looked at making a similar refactoring to this file before - but realised that removing this `resourceStatusCheckIsEnabled` condition means that we are prone to log out messages for all the other conditions that we check.

The verbose log (line 52) is probably much of a muchness, but the warning (line 58) might cause some frustration to customers.

It feels like yelling at people to use a release newer than 2 years old is probably ok at this point though.
I'm alright with risking a complaint or two if you are.

Reply:
Do we try and get them anywhere else aside from https://github.com/OctopusDeploy/Calamari/pull/1544/files#diff-93633c3f4bed8f073fc29d72d49a09cd76ce0175d358713855b9f7ff4000b1fdL67?

In the old version that doesn't run run if `resourceStatusCheckIsEnabled` is false

Reply:
It does because of the logical `||`